src/vnsw/agent/contrail/SConscript

```diff
@@ -104,8 +104,7 @@ if sys.platform.startswith('freebsd'):
 elif sys.platform.startswith('linux'):
     platform_dependent = 'linux/pkt0_interface.cc'
 elif sys.platform.startswith('win'):
-    platform_dependent = ['windows/pkt0_interface.cc','#build/lib/windowsstubs.lib']
-
+    platform_dependent = ['windows/pkt0_interface.cc','#build/lib/windowsstubs.lib', env.RES('windows/resource.rc')]
```

Comment:
What's going to happen when someone modifies resource.h? Shall we add resource.h to platform_dependent files?